creator-node/src/snapbackSM.js

```diff
@@ -124,19 +134,38 @@ class SnapbackSM {
     }, {})
   }
 
-  // Enqueue a sync request to a particular secondary
-  async issueSecondarySync (userWallet, secondaryEndpoint, primaryEndpoint, primaryClockValue) {
+  // Enqueues a manual (high priority) sync.
+  // Returns the added job.
+  async enqueueManualSync ({ userWallet, secondaryEndpoint, primaryEndpoint }) {
```

Comment:
can we have primaryEndpoint first (if order doesnt matter)

Reply:
will do